README.md

```diff
@@ -65,11 +66,30 @@ make up
 
 For more information, check out the documentation in the [presto_cluster](./example/presto_cluster) README.
 
+### Connect to the services (proxies)
+Since the services in this module uses the [`sidecar_service`](https://www.nomadproject.io/docs/job-specification/sidecar_service), you need to connect to the sevices using a [consul connect proxy](https://www.consul.io/commands/connect/proxy).
```

Comment:
```suggestion
Since the services in this module use the [`sidecar_service`](https://www.nomadproject.io/docs/job-specification/sidecar_service), you need to connect to the services using a [consul connect proxy](https://www.consul.io/commands/connect/proxy).
```